<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(91, 91, 91);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>441</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="2" column="1">
      <widget class="QPushButton" name="N8">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2E3B4E;
    color: white;
    border-radius: 10px;
    font-size: 16px;
    padding: 5px;
    border: 2px solid #555;
    box-shadow: 2px 2px 5px rgba(0,0,0,0.3);
}</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="N0">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2E3B4E;
    color: white;
    border-radius: 10px;
    font-size: 16px;
    padding: 5px;
    border: 2px solid #555;
    box-shadow: 2px 2px 5px rgba(0,0,0,0.3);
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="N6">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2E3B4E;
    color: white;
    border-radius: 10px;
    font-size: 16px;
    padding: 5px;
    border: 2px solid #555;
    box-shadow: 2px 2px 5px rgba(0,0,0,0.3);
}</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="N5">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2E3B4E;
    color: white;
    border-radius: 10px;
    font-size: 16px;
    padding: 5px;
    border: 2px solid #555;
    box-shadow: 2px 2px 5px rgba(0,0,0,0.3);
}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="N3">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2E3B4E;
    color: white;
    border-radius: 10px;
    font-size: 16px;
    padding: 5px;
    border: 2px solid #555;
    box-shadow: 2px 2px 5px rgba(0,0,0,0.3);
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="add0">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2E3B4E;
    color: white;
    border-radius: 10px;
    font-size: 16px;
    padding: 5px;
    border: 2px solid #555;
    box-shadow: 2px 2px 5px rgba(0,0,0,0.3);
}</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="mul2">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2E3B4E;
    color: white;
    border-radius: 10px;
    font-size: 16px;
    padding: 5px;
    border: 2px solid #555;
    box-shadow: 2px 2px 5px rgba(0,0,0,0.3);
}</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="N9">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2E3B4E;
    color: white;
    border-radius: 10px;
    font-size: 16px;
    padding: 5px;
    border: 2px solid #555;
    box-shadow: 2px 2px 5px rgba(0,0,0,0.3);
}</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="enter">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2A9D8F;
    color: white;
    border-radius: 10px;
    font-size: 16px;
    padding: 5px;
    border: 2px solid #555;
    box-shadow: 2px 2px 5px rgba(0,0,0,0.3);
}</string>
       </property>
       <property name="text">
        <string>enter</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="clear">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #E63946;
    color: white;
    border-radius: 10px;
    font-size: 16px;
    padding: 5px;
    border: 2px solid #555;
    box-shadow: 2px 2px 5px rgba(0,0,0,0.3);
}</string>
       </property>
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="N2">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2E3B4E;
    color: white;
    border-radius: 10px;
    font-size: 16px;
    padding: 5px;
    border: 2px solid #555;
    box-shadow: 2px 2px 5px rgba(0,0,0,0.3);
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="div3">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2E3B4E;
    color: white;
    border-radius: 10px;
    font-size: 16px;
    padding: 5px;
    border: 2px solid #555;
    box-shadow: 2px 2px 5px rgba(0,0,0,0.3);
}</string>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="sub1">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2E3B4E;
    color: white;
    border-radius: 10px;
    font-size: 16px;
    padding: 5px;
    border: 2px solid #555;
    box-shadow: 2px 2px 5px rgba(0,0,0,0.3);
}</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="N4">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2E3B4E;
    color: white;
    border-radius: 10px;
    font-size: 16px;
    padding: 5px;
    border: 2px solid #555;
    box-shadow: 2px 2px 5px rgba(0,0,0,0.3);
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="N7">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2E3B4E;
    color: white;
    border-radius: 10px;
    font-size: 16px;
    padding: 5px;
    border: 2px solid #555;
    box-shadow: 2px 2px 5px rgba(0,0,0,0.3);
}</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="N1">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2E3B4E;
    color: white;
    border-radius: 10px;
    font-size: 16px;
    padding: 5px;
    border: 2px solid #555;
    box-shadow: 2px 2px 5px rgba(0,0,0,0.3);
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Number2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Number1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Result">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    border: 2px solid gray;
    border-radius: 5px;
    font-size: 24px; /* Suurempi fontti */
    text-align: center; /* Keskittää numerot */
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="num2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    border: 2px solid gray;
    border-radius: 5px;
    font-size: 24px; /* Suurempi fontti */
    text-align: center; /* Keskittää numerot */
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="num1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    border: 2px solid gray;
    border-radius: 5px;
    font-size: 24px; /* Suurempi fontti */
    text-align: center; /* Keskittää numerot */
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
